cluster-autoscaler/provisioningrequest/checkcapacity/processor.go

```diff
@@ -47,7 +49,7 @@ func NewCheckCapacityProcessor() *checkCapacityProcessor {
 // -BookingExpired condition for Provisioned ProvisioningRequest if capacity reservation time is expired.
 // -Failed condition for ProvisioningRequest that were not provisioned during defaultExpirationTime.
 // TODO(yaroslava): fetch reservation and expiration time from ProvisioningRequest
-func (p *checkCapacityProcessor) Process(provReqs []*provreqwrapper.ProvisioningRequest) {
+func (p *checkCapacityProcessor) Process(client *provreqclient.ProvisioningRequestClient, provReqs []*provreqwrapper.ProvisioningRequest) {
```

Comment:
I think we should make the client part of the `checkCapacityProcessor` struct and do not pass it here.

Reply:
+1